src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -1231,6 +1353,23 @@ subroutine initialize_dyn_split_RK2(u, v, h, uh, vh, eta, Time, G, GV, US, param
       'Zonal Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
   CS%id_PFv = register_diag_field('ocean_model', 'PFv', diag%axesCvL, Time, &
       'Meridional Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_hfPFu = register_diag_field('ocean_model', 'hf_PFu', diag%axesCuL, Time, &
+      'Thickness-weighted Zonal Pressure Force Acceleration', 'm s-2', v_extensive=.true., conversion=US%L_T2_to_m_s2)
+  CS%id_hfPFv = register_diag_field('ocean_model', 'hf_PFv', diag%axesCvL, Time, &
+      'Thickness-weighted Meridional Pressure Force Acceleration', 'm s-2', v_extensive=.true., conversion=US%L_T2_to_m_s2)
+  CS%id_hfCAu = register_diag_field('ocean_model', 'hf_CAu', diag%axesCuL, Time, &
+      'Thickness-weighted Zonal Coriolis and Advective Acceleration', 'm s-2', v_extensive=.true., conversion=US%L_T2_to_m_s2)
+  CS%id_hfCAv = register_diag_field('ocean_model', 'hf_CAv', diag%axesCvL, Time, &
+      'Thickness-weighted Meridional Coriolis and Advective Acceleration', 'm s-2', v_extensive=.true., &
+      conversion=US%L_T2_to_m_s2)
+  CS%id_hfPFu_2d = register_diag_field('ocean_model', 'hf_PFu_2d', diag%axesCu1, Time, &
+      'Barotropic Thickness-weighted Zonal Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_hfPFv_2d = register_diag_field('ocean_model', 'hf_PFv_2d', diag%axesCv1, Time, &
+      'Barotropic Thickness-weighted Meridional Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_hfCAu_2d = register_diag_field('ocean_model', 'hf_CAu_2d', diag%axesCu1, Time, &
+      'Barotropic Thickness-weighted Zonal Coriolis and Advective Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_hfCAv_2d = register_diag_field('ocean_model', 'hf_CAv_2d', diag%axesCv1, Time, &
+      'Barotropic Thickness-weighted Meridional Coriolis and Advective Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
```

Comment:
Similar to above, the variable id name and the diagnostic name should be the same.